src/mds/Server.cc

```diff
@@ -5722,6 +5722,32 @@ int Server::parse_layout_vxattr(string name, string value, const OSDMap& osdmap,
   return 0;
 }
 
+int64_t Server::return_bytes(string& value_str)
+{
+  if (value_str.length() < 1) {
+    return -CEPHFS_EINVAL;
+  }
```

Comment:
nit: the empty string can simply be tested as
```
if (value_str.empty()) {
  return -CEPHFS_EINVAL;
}
```

it's much easier on the eyes

Reply:
done. thanks.